receiver/splunkenterprisereceiver/README.md

```diff
@@ -1,5 +1,38 @@
 # Splunk Enterprise Receiver
----
 
-The Splunk Enterprise Receiver is a pull based tool which enables the ingestion of key performance metrics (KPI's) describing the operational status of a user's Splunk Enterprise deployment to be 
-added to their OpenTelemetry Pipeline.
+The Splunk Enterprise Receiver is a pull based tool which enables the ingestion of performance metrics describing the operational status of a user's Splunk Enterprise deployment to an appropriate observability tool.
```

Comment:
LGTM. Can we add the generated yaml entry for READMEs in a separate PR? see https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/azureblobreceiver/README.md?plain=1#L3 for example. Just add those comments and generate via mdatagen.

Reply:
got it